node-require/calculate.js

```diff
@@ -0,0 +1,20 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const divide = require('./divide');
+const multiply = require('./multiply');
+
+if (process.argv[3] === ('plus' || '+')) {
+  const sum = add(parseFloat(process.argv[2]), parseFloat(process.argv[4]));
+  console.log(`${process.argv[2]} + ${process.argv[4]} = ${sum}`);
+} else if (process.argv[3] === ('minus' || '-')) {
+  const difference = subtract(parseFloat(process.argv[2]), parseFloat(process.argv[4]));
+  console.log(`${process.argv[2]} - ${process.argv[4]} = ${difference}`);
+} else if (process.argv[3] === ('divide' || '/')) {
+  const quotient = divide(parseFloat(process.argv[2]), parseFloat(process.argv[4]));
+  console.log(`${process.argv[2]} / ${process.argv[4]} = ${quotient}`);
+} else if (process.argv[3] === ('multiply' || '*')) {
+  const multiplied = multiply(parseFloat(process.argv[2]), parseFloat(process.argv[4]));
+  console.log(`${process.argv[2]} x ${process.argv[4]} = ${multiplied}`);
```

Comment:
To avoid a bunch of repetition it would be best to save `process.argv[2]`, `process.argv[3]`, and `process.argv[4]` into variables. Also when you create the variables you can parse the number so you don't have to do it in 8 different places. For example:

```JAVASCRIPT
const num1 = parseFloat(process.argv[2]);
const operator = process.argv[3];
const num2 = parseFloat(process.argv[4]);
```